classes/core.php

```diff
@@ -1447,9 +1470,40 @@ static public function run_calculation($value, $field, $form)
 			return new WP_Error($field['ID'] . '-calculation',
 				__('Calculation is invalid (division by zero)', 'caldera-forms'));
 		}
+	
+		//If PHP version is less than 7.2.0, continue using old function
+		if(phpversion() < '7.2.0'){
+
+			$total_function = create_function(null, 'return ' . $formula . ';');
+			$total = $total_function();
+
+		} else { //avoid using create_function() when using PHP version >= 7.2.0
+
+			//Check if the formula use a symbol or function not supported by the parser but declared as supported at https://calderaforms.com/doc/calculation-fields/
```

Comment:
@New0 All of this inline code explaining WHY you did the logic in this order is great. Helps with code review and I'm sure it will help us later.